diff --git a/node_modules/cypress/lib/tasks/download.js b/node_modules/cypress/lib/tasks/download.js
index ca4e557..8dc071d 100644
--- a/node_modules/cypress/lib/tasks/download.js
+++ b/node_modules/cypress/lib/tasks/download.js
@@ -52,6 +52,7 @@ const getCA = () => {
 const prepend = (arch, urlPath, version) => {
   const endpoint = url.resolve(getBaseUrl(), urlPath);
   const platform = os.platform();
+  if (platform === 'win32') { arch = 'x64' };
   const pathTemplate = util.getEnv('CYPRESS_DOWNLOAD_PATH_TEMPLATE', true);
   return pathTemplate ? pathTemplate.replace(/\\?\$\{endpoint\}/g, endpoint).replace(/\\?\$\{platform\}/g, platform).replace(/\\?\$\{arch\}/g, arch).replace(/\\?\$\{version\}/g, version) : `${endpoint}?platform=${platform}&arch=${arch}`;
 };
diff --git a/node_modules/cypress/lib/tasks/install.js b/node_modules/cypress/lib/tasks/install.js
index 9e15a2c..93c897a 100644
--- a/node_modules/cypress/lib/tasks/install.js
+++ b/node_modules/cypress/lib/tasks/install.js
@@ -133,8 +133,9 @@ const downloadAndUnzip = ({
   return Promise.resolve(tasks.run());
 };
 const validateOS = () => {
+  debug('patched for win32-arm64')
   return util.getPlatformInfo().then(platformInfo => {
-    return platformInfo.match(/(win32-x64|linux-x64|linux-arm64|darwin-x64|darwin-arm64)/);
+    return platformInfo.match(/(win32-x64|win32-arm64|linux-x64|linux-arm64|darwin-x64|darwin-arm64)/);
   });
 };
 
